### PR TITLE
Edit notice text for PP, Copyright & Honor Code

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd the copyright link isn't working, it's probably a config issue

### CODE REVIEW

**Comment by caesar2164** [comment]
[Inconsistent messaging on TOS/PP/HONOR pages](https://trello.com/c/hhvqvMb9/166-inconsistent-messaging-on-tos-pp-honor-pages)

File: lagunita/lms/templates/static_templates/copyright.html [hunk 1]
```diff
@@ -24,13 +24,6 @@
 
 <main id="main" aria-label="Content" tabindex="-1">
     <section class="container about tos">
-        <h2><em>
```
Comment (stvstnfrd) [inline lagunita/lms/templates/static_templates/copyright.html:27]: SUclass doesn't have a copyright file?

Comment (caesar2164) [reply lagunita/lms/templates/static_templates/copyright.html:27]: As in this file shouldn't be there?

**Comment by stvstnfrd** [comment]
Please search this repo for all occurrences and update them all accordingly.

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**